dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports-reconciliation-summary.controller.test.js

```diff
@@ -57,6 +64,49 @@ describe('getUtilisationReportsReconciliationSummary', () => {
         bank: { id: MOCK_BANKS.BARCLAYS.id, name: MOCK_BANKS.BARCLAYS.name },
         status: UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED,
       },
+    ]);
+  });
+
+  it('returns the reconciliation summary for all banks', async () => {
+    // Arrange
+    const { req, res } = getHttpMocks();
+
+    const banks = [MOCK_BANKS.BARCLAYS, MOCK_BANKS.HSBC];
+    getAllBanks.mockResolvedValue(banks);
+
+    const barclaysReport = {
+      ...MOCK_UTILISATION_REPORT,
+      bank: { id: MOCK_BANKS.BARCLAYS.id, name: MOCK_BANKS.BARCLAYS.name },
+      status: UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_IN_PROGRESS,
+    };
+    getUtilisationReportDetailsByBankIdMonthAndYear.mockImplementation(async (bankId) => {
+      switch (bankId) {
+        case MOCK_BANKS.BARCLAYS.id:
+          return barclaysReport;
+        case MOCK_BANKS.HSBC.id:
```

Comment:
If there is no specific behaviour for the HSBC case, why not just capture that in the `default`?